docs/core/idioms/targets.md

```diff
@@ -0,0 +1,63 @@
+# Using Result targets for efficient caching
+
+Targets in Prefect are templatable location strings that are used to check for the existence of a task [Result](/core/concepts/results.html). This is useful in cases where you might want a task to only write data to a location once or merely not rerun if some piece of data is present. If a result exists at that location then the task run will enter a cached state. This behavior is commonly referred to as caching in Prefect and it is generally used when you do not want to recompute the result of a task if it already exists.
+
+The flow below will write the result of the first task to a target specified with a filepath of `task_name-today`. For a list of all available context variables at runtime visit the [context API documentation](/api/latest/utilities/context.html#context). This is a powerful construct because it means that only the first run of this task on any given day will run and write the result. Any other runs up until the next calendar day will use the cached result found at `task_name-today`.
+
+::: warning Result Types
+Please review the [documentation for each result type](/api/latest/engine/results.html) before you configure. Some types have specific configuration options for where to write the result.
+
+For example: the `LocalResult` below has an optional `dir` kwargs which accepts an absolute path to a directory for storing results.
```

Comment:
Can you add a sentence to this local result call out that says "In this case the `target` value will be appended to the directory of choice."